docker-compose/docker-compose.yml

```diff
@@ -51,7 +51,7 @@ services:
 
   # WHISK CONTROLLER
   controller:
-    image: whisk/controller:latest
+    image: openwhisk/controller:6a56c45
```

Comment:
Regarding a static version I see 2 options:
1. keeping a static version ensures `quick-start` runs correctly all the time. The question is how often we upgrade the version.
2. going with `latest` version would keep OW updated all the time, but the `quick-start` might fail when OW configuration updates.

What should we pick here ?